src/backend/common/deferred/clients/fake_client.py

```diff
@@ -1,4 +1,46 @@
 from backend.common.deferred.clients.rq_client import RQTaskClient
+from backend.common.deferred.queues.rq_queue import RQTaskQueue
+from backend.common.deferred.tasks.task import Task
+from backend.common.redis import RedisClient
+
+
+class InlineRQTaskQueue(RQTaskQueue):
```

Comment:
nit: Can we move this to live with the rest of the queues?

Reply:
sure, I can move them